Cargo.toml

```diff
@@ -29,7 +27,7 @@ moxin-protocol = { path = "moxin-protocol" }
 moxin-backend = { path = "moxin-backend" }
 moxin-fake-backend = { path = "moxin-fake-backend" }
 
-makepad-widgets = { git = "https://github.com/jmbejar/makepad", branch = "moxin-release-v1" }
```

Comment:
We need to revert this before merging.

But cherry picking the toggle panel commits on top of `moxin-release-v1` doesn't work. In that branch, the `import View` inside toggle panel causes a stack overflow (similar to what would happen in the `rik` branch if you try to import something from `base` inside toggle panel.

We may need to update moxin-release-v1 with stuff from rik.